hack/e2e.sh

```diff
@@ -68,7 +68,12 @@ Examples:
 
   Note that current kubetest needs root permission to cleanup.
 
-3) To run against cluster with GCE provider locally
+3) TO run against a cluster started by kind
```

Comment:
Can you add a note about the limitation with block devices here?

Reply:
done